core/templates/css/oppia-material.css

```diff
@@ -0,0 +1,2058 @@
+/* Colors for the ripple elements.*/
```

Comment:
Could you please attach the doc as well describing how this css is generated? (The one that you shared with the team)

Reply:
Done here: https://github.com/oppia/oppia/pull/9957#issue-452435671